dev/base_include

```diff
@@ -192,7 +192,7 @@ let qid = Libnames.qualid_of_string;;
 (* parsing of terms *)
 
 let parse_constr = Pcoq.parse_string Pcoq.Constr.constr;;
-let parse_tac    = Pcoq.parse_string Ltac_plugin.Pltac.tactic;;
+(*let parse_tac    = Pcoq.parse_string Ltac_plugin.Pltac.tactic;;*)
```

Comment:
Sorry, this may be a stupid question but is this intended? When putting code in comments without a comment explaining why, it's generally that the author forgot to uncomment at some point.